README.md

```diff
@@ -41,33 +41,33 @@ Good news, you don't have to!  Enter log4cats!  Read on!
 import org.typelevel.log4cats.Logger
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 import cats.effect.Sync
-import cats.implicits._
+import cats.implicits.*
 
 object MyThing {
   // Impure But What 90% of Folks I know do with log4s
-  implicit def unsafeLogger[F[_]: Sync] = Slf4jLogger.getLogger[F]
+  implicit def unsafeLogger[F[_] : Sync] = Slf4jLogger.getLogger[F]
```

Comment:
Is there a warning on these spaces?  This is the thing that seems strangest to me.

Reply:
The default `spaces.beforeContextBoundColon` is [Never](https://scalameta.org/scalafmt/docs/configuration.html#spacesbeforecontextboundcolon) and it doesn't seem [configured](https://github.com/typelevel/log4cats/blob/main/.scalafmt.conf) so it's a bit weird these have been added

Reply:
I'm going to assume that it's something weird with my editor, so I've rolled it back.